packages/apollo/src/util/generate-slug.ts

```diff
@@ -1,9 +1,6 @@
 import { prisma } from "./prisma";
 
-export const generateSlug = async (
-  entity: "branch" | "event",
-  name: string,
-) => {
+export const generateSlug = async (entity: "event", name: string) => {
```

Comment:
Unless there is something I'm missing we probably don't need `entity` at all anymore but :shrug:

Reply:
I mainly did it for ~~TS to stop complaining~~ future-proofing